### ISSUE TITLE
Is the project completely dead?

### ISSUE THREAD

**Issue by elijahcruz12** [davibennun/laravel-raygun#8]
I'm tryintg to use this on my Laravel 8 installation, but I think that because it doesn't state support for Laravel 8, it may not work.

**Comment by hermaxim** [comment]
larave new blog --jet

**Comment by elijahcruz12** [comment]
@hermaxim what would creating a new laravel project have to do with the question asked about an existing project. On top of that, I doubt that raygun would require jetstream (Personally don't prefer anyways) for using Raygun, especially because Jetstream is fairly new and most sites aren't using Jetstream anyways.

**Comment by davibennun** [comment]
@elijahcruz12 Yes, I stopped working on this on Laravel 5, feel free to contribute to the project :)

**Comment by davibennun** [comment]
@elijahcruz12 I can transfer this repo to you if you want to implement support for later laravel versions

**Comment by elijahcruz12** [comment]
I think that could be a great Idea, as honestly the only framework I use is Laravel. The only support nowadays really should be Laravel 7.x and 8.x as those versions have been the latest for quite a while now. Laravel 6.x is virtually unheard of at this point, and Laravel 5.x, while still used, is very much seen as needing to be upgraded to a later version of Laravel.

If you want to transfer the repo to me, I can definetely start working to generate support for the latest versions of Laravel.

**Comment by elijahcruz12** [comment]
Bumping this.

**Comment by hermaxim** [comment]
discúlpame pero yo no escribí eso ni siquiera se que es laravel --jet
